File: lib/core/StdIO.py
#!/usr/bin/env python

import os
from sys import stdout
from time import sleep


def Clear():
    os.system("cls" if os.name == "nt" else "clear")


class Logging:
    @staticmethod
    def Typewriter(text, delay=0.001):
        for char in text:
            stdout.write(char)
            stdout.flush()
            sleep(delay)
        stdout.write("\n")
        stdout.flush()

File: internal/ansi/stdio.go
package ansi

import (
	"fmt"
	"os"
	"os/exec"
	"time"
)

func Clear() {
	Cmd := exec.Command("clear")
	Cmd.Stdout = os.Stdout
	Cmd.Run()
}

func Typewriter(Text string, Delay time.Duration) {
	for _, Ch := range Text {
		fmt.Printf("%c", Ch)
		time.Sleep(Delay)
	}
	fmt.Println()
}

func TypewriterFast(Text string) {
	Typewriter(Text, time.Millisecond)
}
